pubspec.yaml

```diff
@@ -4,8 +4,8 @@ version: 3.3.5
 homepage: https://github.com/mogol/flutter_secure_storage
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
-  flutter: ">=1.20.0 <2.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
+  # flutter: ">=1.24.0 <2.0.0"
```

Comment:
This shouldn't be commented out right?

Reply:
that's right! Sorry, can't say why that's there. It looks wrong.

Reply:
I've changed it to use as a minimum constraint the version of flutter that is on beta channel